### ISSUE TITLE
Support for Security Log Profiles in Virtual Server resource

### ISSUE THREAD

**Issue by Towl** [F5Networks/terraform-provider-bigip#447]
### The feature
I'm working with Ansible and I want to move to Terraform. The module for Ansible allowed us to do something like this:
```yaml
- name: Create VS
  bigip_virtual_server:
    provider: "{{ provider }}"
    state: present
    name: "name"
    description: "exemple"
    source: "0.0.0.0/0"
    destination: "{{ vs_info.ip }}"
    port: "{{ vs_info.port }}"
    profiles: "profile"
    pool: ""
    enabled_vlans:
      - "some-vlan"
    snat: Automap
    policies:
      - "some-policy"
    security_log_profiles:
      - "Log_default_illegal_requests"
    irules:
      - irule-sni
  delegate_to: localhost
```
The important part is the `security_log_profiles` argument. I don't see how to do it with the Terraform module. I search in the existing issues and in the code but nothing about this feature seems to be implemented yet.
For reference, [the documentation is here](https://docs.ansible.com/ansible/latest/collections/f5networks/f5_modules/bigip_virtual_server_module.html#parameter-security_log_profiles).

### Solution
Ideally, an additional property of the resource `bigip_ltm_virtual_server` should be created specifically to handle this Security Log Profiles feature.

### Describe alternatives you've considered
I don't find any using Terraform currently.

**Comment by papineni87** [comment]
Added support for secuity_log_profile to bigip provider ( https://github.com/F5Networks/terraform-provider-bigip/pull/496 )

**Comment by RavinderReddyF5** [comment]
@Towl fixed in v1.10.0, please check and open new issue if u see any problem.
